Directory.Build.props

```diff
@@ -130,7 +126,7 @@
     <RuntimeInstallerBaseName>aspnetcore-runtime</RuntimeInstallerBaseName>
     <TargetingPackInstallerBaseName>aspnetcore-targeting-pack</TargetingPackInstallerBaseName>
 
-    <!-- Produce targeting pack installers/packages once per major.minor. -->
+    <!-- Used to only produce targeting pack installers/packages once per major.minor. -->
```

Comment:
nit - change to "This is used to..." to avoid devs accidentally interpreting this as "In the past, we only produced..."